linux/ibus-keyman/src/engine.c

```diff
@@ -974,17 +971,17 @@ ibus_keyman_engine_focus_out (IBusEngine *engine)
 {
     IBusKeymanEngine *keyman = (IBusKeymanEngine *) engine;
 
-    g_message("ibus_keyman_engine_focus_out");
+    g_message("%s", __FUNCTION__);
     km_kbp_context_clear(km_kbp_state_context(keyman->state));
     parent_class->focus_out (engine);
 }
 
 static void
 ibus_keyman_engine_reset (IBusEngine *engine)
 {
-    g_message("ibus_keyman_engine_reset");
+    g_message("%s", __FUNCTION__);
     parent_class->reset (engine);
-    ibus_keyman_engine_focus_in (engine);
+    reset_context(engine);
```

Comment:
What is the purpose of this change?

Reply:
Both `ibus_keyman_engine_reset` and `ibus_keyman_engine_focus_in` are methods that get called by ibus. Having one call the other makes it much harder to see from a log file what methods ibus actually calls.

Reply:
Great, that makes sense, thanks!